seed/static/seed/js/services/column_mappings_service.js

```diff
@@ -16,15 +16,15 @@ angular.module('BE.seed.service.column_mappings', []).factory('column_mappings_s
       if (filter_preset_types != null) {
         params.preset_type = filter_preset_types;
       }
-      return $http.get('/api/v2/column_mapping_presets/', {
+      return $http.post('/api/v3/column_mapping_profiles/filter/', params.preset_type, {
```

Comment:
This isn't working for me. As part of any changes for this, let's remove the preset_type from query_params.